meta-arm-bsp/recipes-bsp/u-boot/u-boot_%.bbappend

```diff
@@ -5,6 +5,7 @@ MACHINE_U-BOOT_REQUIRE:corstone1000 = "u-boot-corstone1000.inc"
 MACHINE_U-BOOT_REQUIRE:fvp-base = "u-boot-fvp-base.inc"
 MACHINE_U-BOOT_REQUIRE:juno = "u-boot-juno.inc"
 MACHINE_U-BOOT_REQUIRE:tc = "u-boot-tc.inc"
+MACHINE_U-BOOT_REQUIRE:qemuarm64-secureboot = "${@bb.utils.contains('MACHINE_FEATURES', 'uefi-secureboot', 'u-boot-qemuarm64-secureboot.inc', '', d)}"
```

Comment:
@mikkorapeli-linaro

> I think this should be generic to all machines if uefi-secureboot is in
> MACHINE_FEATURES.
> 
> I know meta-arm will only test qemuarm64-secureboot but users will have different
> machine names and would expect this to work there too.

Make it generic; `uefi-secureboot` would be moved up to another layer, right?

Reply:
I don't see a need for this to be qemuarm64-secureboot specific. Thus just use a config fragment which is tied to the distro/machine feature, and enable the feature on qemuarm64-secureboot.

Reply:
Done